Dockerfile

```diff
@@ -100,5 +100,7 @@ RUN git config --global core.symlinks false
 # Remove setuptools-65.5.1 installed from python:3.11-slim base image as fix for CVE-2024-6345 until image will be updated
 RUN rm -rf /usr/local/lib/python3.11/site-packages/setuptools-65.5.1.dist-info
 
+RUN python -m playwright install --with-deps chromium
```

Comment:
why do we need to install it both in `pyproject.toml` and here?

Reply:
This here is installing the required browsers for playwright to run. This is not done automatically when the python `playwright` dependency is installed.

Similarly to pypandoc, I believe we could do this at runtime but thought it'd be cleaner like this.